conditional/templates/base.html

```diff
@@ -28,8 +28,9 @@
         </div>
         {% endif %}
 
+        <footer>
+            <a class="footer-version" href="https://github.com/ComputerScienceHouse/conditional">Conditional ({{config["GIT_REVISION"]}})</a>
```

Comment:
Can we make the href also include the commit hash? So it links directly to browse at that part?